extra/alacritty-msg.man

```diff
@@ -2,18 +2,27 @@
 .SH NAME
 alacritty-msg \- Send messages to Alacritty
 .SH "SYNOPSIS"
-alacritty msg [OPTIONS] [MESSAGES]
+alacritty msg [OPTIONS] [MESSAGE] [MESSAGE_OPTIONS]
 .SH DESCRIPTION
 This command communicates with running Alacritty instances through a socket,
 making it possible to control Alacritty without directly accessing it.
 .SH "OPTIONS"
-.TP
 \fB\-s\fR, \fB\-\-socket\fR <socket>
 Path for IPC socket creation
 .SH "MESSAGES"
-.TP
 \fBcreate-window\fR
 Create a new window in the same Alacritty process
+.TP
+.SH "OPTIONS"
```

Comment:
Actually one more thing: This is on the same level as `create-window` at the moment. I think it would be better to indent it one more level? That way nobody thinks that `OPTIONS` is a valid message to send?